src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ReleaseProcessorService.cs

```diff
@@ -37,32 +37,20 @@ public class ReleaseProcessorService : IReleaseProcessorService
             Thread.Sleep(new Random().Next(1, 5) * 1000);
             
             var release = CreateOrUpdateRelease(message, context);
-            var subject = RemoveAndCreateSubject(message.SubjectId, subjectData.Name, release, context);
+            RemoveSubject(message.SubjectId, subjectData.Name, release, context);
             
+            var subject = CreateSubject(message.SubjectId, subjectData.Name, release, context);
             CreateReleaseFileLink(message, contentDbContext, release, subject);
-            CreateReleaseSubjectLink(context, release, subject);
             
             return subject;
         }
 
-        private static void CreateReleaseSubjectLink(StatisticsDbContext context, Release release, Subject subject)
-        {
-            context
-                .ReleaseSubject
-                .Add(new ReleaseSubject
-                {
-                    ReleaseId = release.Id,
-                    SubjectId = subject.Id
-                });
-        }
-
         private static void CreateReleaseFileLink(ImportMessage message, ContentDbContext contentDbContext, Release release,
             Subject subject)
         {
             var releaseFileLink = contentDbContext
                 .ReleaseFiles
-                .Include(
-                    f => f.ReleaseFileReference)
+                .Include(f => f.ReleaseFileReference)
```

Comment:
Do you think this `.Include` is necessary or is it clever because it's mentioned in `.FirstOrDefault` as `f.ReleaseFileReference.Filename`?

I'm just asking because there is a very similar query below in this PR which uses `Subject.Name` in `.FirstOrDefault`  but doesn't have `.Include(r => r.Subject)`

Reply:
Oddly yes you do need the include but not in the second case - not entirely sure why!